### ISSUE TITLE
Deploy to Azure is broken

### ISSUE THREAD

**Issue by qbonie1984** [rauchg/slackin#304]
Hi, The deployment is not throwing any error but when I click on the URL in the deployment status, I get redirected to generic azure landing page. Help please! thanks!

**Comment by marchofreason** [comment]
I'm having this problem as well. Showing up a page which says 

"Your App Service app has been created
Go to your app's Quick Start guide in the Azure portal to get started or read our deployment documentation."

Anyone else have any luck with this?

**Comment by andyrblank** [comment]
It's running hostingstart.html. If I go into "Application Settings" in Azure I can remove that as a default document but I'm not sure which file should be the start page.

**Comment by andyrblank** [comment]
I'm still investigating but it looks like the web.config file is not getting deployed.

**Comment by andyrblank** [comment]
FYI: In my App Service if I scroll down to App Service Editor (preview) I can right-click and add a new file so I added the web.config and copied the contents from this repository and now I'm getting an error returned to check my API key so I think it is working and I need to correct my API key.

**Comment by andyrblank** [comment]
Ok so when I started this I was working with a friend who gave me a token and team name that I couldn't verify. So for testing I built my own and I have confirmed that by deploying a web.config and removing hostingstart.html from the default documents that it is working!

**Comment by jpoon** [comment]
Yeah, as @andyrblank, it's missing the `web.config`. Go to the Azure portal, open the console (either through Kudu or right through the settings page for the app service), then in `wwwroot` do: `curl -O https://raw.githubusercontent.com/rauchg/slackin/master/web.config`

**Comment by Rogerbut** [comment]
When I went to application settings in Azure i found that index.aspx (the name of my home page) was NOT included in the default documents - adding that fixed the issue for me
